ci/infra/testrunner/tests/test_node_reboot.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import json
+import time
+
+import pytest
+from timeout_decorator import timeout
+
+logger = logging.getLogger("testrunner")
+
+
+@timeout(120)
+def wait_for_node_reboot(node_type, node_number, platform):
+    while True:
+        try:
+            platform.ssh_run(node_type, node_number, 'echo hello')
+        except Exception as ex:
+            logger.debug(ex)
+        else:
+            break
+
+        time.sleep(5)
+
+
+@timeout(120)
+def check_nodes_ready(kubectl):
```

Comment:
I think this function can be useful for other tests. I would consider to make it part of the test library maybe in a new `testlib.py` file